pkg/labeler/labeler.pb.go

```diff
@@ -1,155 +1,300 @@
+//
+//Copyright 2019 The Kubernetes Authors.
```

Comment:
2021?

Reply:
This is autogenerated file. The year is copied from `labeler.proto`. I don't think it makes sense to modify the proto file itself just to update the copyright year